js/PulsarUIComponent.js

```diff
@@ -64,31 +64,66 @@ PulsarUIComponent.getDatatableOptions = function ($table) {
         buttons: [],
         className: 'dt-row-selected',
         columnDefs: [
-            { 
-                className: 'control', 
-                orderable: false, 
+            {
+                className: 'control',
+                orderable: false,
                 targets: 0
             },
-            { 
-                searchable: false, 
+            {
+                searchable: false,
                 targets: [0]
             },
             {
                 orderable: false,
                 targets: [0, 1]
             }
         ],
+        initComplete: function () {
+            let pagination = $(this).closest('.dataTables_wrapper').find('.dataTables_paginate');
+
+            // Wrap pagination in nav
+            pagination.wrap('<nav aria-label="Table pagination"></nav>');
+        },
+        drawCallback: function() {
+            let pagination = $(this).closest('.dataTables_wrapper').find('.dataTables_paginate'),
+                paginationNumbers;
+
+            // Only show pagination when needed
+            pagination.toggle(this.api().page.info().pages > 1);
+
+            // Add aria-current to current page number
+            pagination.find('.paginate_button.current').attr('aria-current', 'true');
+
+            // Remove disable links
+            pagination.find('.paginate_button.disabled').addClass('u-display-none');
+
+            // Add aria-labels to numbered links
+            paginationNumbers = pagination.find('.paginate_button:not(.first, .previous, .next, .last)');
+            paginationNumbers.each(function () {
+                let $numberLink = $(this);
+                $numberLink.attr('aria-label', 'Page ' + $numberLink.text());
+            });
+        },
         dom: dom,
         language: {
             emptyTable: langEmptyTable,
             info: "Showing _START_ to _END_ of _TOTAL_ items",
             infoEmpty: 'No items',
             infoFiltered: " (filtered from _MAX_ items)",
             zeroRecords: "No items matched your filter, please clear it and try again",
-            search: "Filter records"
+            search: "Filter records",
+            aria: {
+                paginate: {
+                    first:    'First page',
+                    previous: 'Previous page',
+                    next:     'Next page',
+                    last:     'Last page'
+                }
+            }
         },
-        lengthChange: false,
+        lengthChange: true,
```

Comment:
This will need flagging up in the release notes as a change to expected behaviour, we may need to add the ability for it to be disabled via a data-attribute.